### ISSUE TITLE
Cannot create project: The changes are too large.

### ISSUE THREAD

**Issue by danieltomasz** [typst/webapp-issues#426]
### Description

I just started a subscription  of Typst Pro, I try to clone github repository with the article I am working on, but I keep getting this info
`Cannot create project: The changes are too large.`
the repository is 100mb with mostly markdown files (before I migrated to typst) and some  other files I am using for this project.

**Comment by danieltomasz** [comment]
Repo is private, I can provide access to someone trying to reporoduce

**Comment by laurmaedje** [comment]
The current limit for Git repository size that can be synced is quite low (10MB). We can definitely raise the limit, but I can't yet say by how much. We'll look into that.

**Comment by danieltomasz** [comment]
@laurmaedje  thanks for the info! I was thinking  about moving my full phd thesis to typst (as it offering great app and possibility for others for comments) but with current repo size limitations I will stay with pandoc, markdown and github; 

though I will still continue to test typst within this one project without git/with manual sync with  external git repository (a few big figures in paper might exhaust the  10 mb limit)

**Comment by laurmaedje** [comment]
Yeah, we're aware that the limit as it is now is quick to exhaust. As the feature is still new and experimental, we're a bit cautious and observing how it behaves in production before allowing larger repositories.

**Comment by danieltomasz** [comment]
@laurmaedje would you allow to `git clone --depth 1 ` to add only recent version?

**Comment by danieltomasz** [comment]
Do you have now or plan in the future any beta program for testing?

**Comment by laurmaedje** [comment]
We have increased the repository size limit to 100MB. 

However, our backend needs to clone the full repository to be able to perform the diffing necessary to coalesce local and remote changes. With larger repositories, push and pull time can thus be slow.

**Comment by danieltomasz** [comment]
thanks,  this is really useful and set the reasonable limit!
